### ISSUE TITLE
"ld: warning: directory not found for option" with Framework project with multiple (platform) targets

### ISSUE THREAD

**Issue by fguchelaar** [CocoaPods/CocoaPods#4920]
I'm creating a 'Universal' Framework project, that I want to use in a Workspace with both an iOS and a tvOS App. The project has two targets; one for each platform. I also want to use some Pods in the Framework. But when I add Pods to a Framework project for both targets, I get build-warnings and a linker error.

```
ld: warning: directory not found for option '-F/Users/...../Pods-MultiTarget_Framework_tvOS'`

....

ld: framework not found AFNetworking
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Somehow the Pod-frameworks can not be found when Pods are added to multiple targets within the same Framework-project.

To isolate the issue, from potential other ones, I've created a dedicated Workspace/Project, which can be found here: https://github.com/fguchelaar/MultiTargetFramework 

My Podfile:

```bash
use_frameworks!

workspace 'MultiTarget'

target :MultiTarget_Framework_iOS do
  project 'MultiTarget_Framework/MultiTarget_Framework.xcodeproj'
  platform :ios, "9.0"
  pod "AFNetworking"
  pod "SDWebImage"
end

target :MultiTarget_Framework_tvOS do
  project 'MultiTarget_Framework/MultiTarget_Framework.xcodeproj'
  platform :tvos, "9.1"
  pod "AFNetworking"
  pod "SDWebImage"
end
```

I'm using Cocoapods v1.0.0.beta.3

The Framework-project was pretty much set up according to [this guide](https://medium.com/@ataibarkai/create-a-universal-swift-framework-for-ios-os-x-watchos-and-tvos-2aa26a8190dc#.jqywd1jpk).

**Comment by segiddins** [comment]
@mrackwitz open PR ought to fix this.

**Comment by fguchelaar** [comment]
Anything I can do to help validate?

**Comment by fguchelaar** [comment]
Resolved in 1.0.0.beta.4. Thanks
